public/locales/tr/send.ftl

```diff
@@ -15,6 +16,7 @@ encryptingFile = Şifreleniyor...
 decryptingFile = Şifre çözülüyor...
```

Comment:
Also spotted a few of these... I think @flodolo generally recommends a single `…` entity versus 3 separate `...`.
I can manually fix those up in a separate PR or commit if anybody wants it.

A quick `git grep "\.\.\." public/locales/*/*.ftl` shows about 51 instances:

```
public/locales/cy/send.ftl:encryptingFile = Wrthi'n amgryptio...
public/locales/de/send.ftl:decryptingFile = Wird entschlüsselt...
public/locales/dsb/send.ftl:importingFile = Importěrujo se...
public/locales/dsb/send.ftl:verifyingFile = Pśespytujo se...
public/locales/dsb/send.ftl:encryptingFile = Koděrujo se...
public/locales/dsb/send.ftl:decryptingFile = Dešifrěrujo se...
public/locales/es-ES/send.ftl:importingFile = Imporando...
public/locales/es-ES/send.ftl:verifyingFile = Comprobando...
public/locales/es-ES/send.ftl:encryptingFile = Encriptando...
public/locales/es-ES/send.ftl:decryptingFile = Desencriptando...
public/locales/es-MX/send.ftl:importingFile = Importando...
public/locales/es-MX/send.ftl:verifyingFile = Verificando...
public/locales/es-MX/send.ftl:encryptingFile = Encriptando...
public/locales/es-MX/send.ftl:decryptingFile = Desencriptando...
public/locales/hsb/send.ftl:importingFile = Importuje so...
public/locales/hsb/send.ftl:verifyingFile = Přepruwuje so...
public/locales/hsb/send.ftl:encryptingFile = Zaklučuje so...
public/locales/hsb/send.ftl:decryptingFile = Dešifruje so...
public/locales/it/send.ftl:encryptingFile = Crittazione in corso...
public/locales/it/send.ftl:decryptingFile = Decrittazione in corso...
public/locales/ja/send.ftl:importingFile = インポート中...
public/locales/ja/send.ftl:verifyingFile = 検証中...
public/locales/ja/send.ftl:encryptingFile = 暗号化中...
public/locales/ja/send.ftl:decryptingFile = 復号化中...
public/locales/kab/send.ftl:importingFile = Akter...
public/locales/kab/send.ftl:verifyingFile = Asenqed...
public/locales/kab/send.ftl:encryptingFile = Awgelhen...
public/locales/kab/send.ftl:decryptingFile = Azmek...
public/locales/ms/send.ftl:verifyingFile = Mengesahkan...
public/locales/ms/send.ftl:encryptingFile = Mengenkripsi...
public/locales/ms/send.ftl:decryptingFile = Mengenkripsi...
public/locales/nb-NO/send.ftl:verifyingFile = Verifiserer...
public/locales/nb-NO/send.ftl:encryptingFile = Krypterer...
public/locales/nb-NO/send.ftl:decryptingFile = Dekrypterer...
public/locales/nn-NO/send.ftl:decryptingFile = Dekrypterer...
public/locales/pt-BR/send.ftl:encryptingFile = Criptografando...
```